webhook_server/tests/e2e/server_utils.py

```
@@ -0,0 +1,215 @@
+def start_docker_compose(docker_compose_file: str, project_root: str) -> None:
+    """Start docker-compose container for the webhook server.
+
+    Args:
+        docker_compose_file: Path to docker-compose.yaml file
+        project_root: Project root directory path
+
+    Raises:
+        E2EInfrastructureError: If docker-compose fails to start
+    """
+    logger.info("Building and starting docker-compose container...")
+    result = subprocess.run(
```

Ruff (0.14.5)

`subprocess` call: check for execution of untrusted input

(S603)